Bundle/CoreBundle/Entity/Link.php

```diff
@@ -40,6 +41,8 @@ class Link
      * @var string
      *
      * @ORM\Column(name="url", type="string", length=255, nullable=true)
+     *
+     * @Assert\Url(message = "This value is not a valid URL.")
```

Comment:
Please use a translation string

Reply:
This string already has associated transaltions in the Validator component of Symfony.

Reply:
If it's useless because it's the default value, in this case, you shouldn't set it:
```diff
-@Assert\Url(message = "This value is not a valid URL.")
+@Assert\Url()
```